contracts/modules/TransferManager/GeneralTransferManager.sol

```diff
@@ -300,10 +331,10 @@ contract GeneralTransferManager is GeneralTransferManagerStorage, TransferManage
     /**
      * @notice Internal function used to check whether the investor is in the whitelist or not
             & also checks whether the KYC of investor get expired or not
-     * @param _investor Address of the investor
+     * @param _expiryTime Expiry time of the investor
      */
-    function _onWhitelist(address _investor) internal view returns(bool) {
-        return (whitelist[_investor].expiryTime >= uint64(now)); /*solium-disable-line security/no-block-members*/
+    function _onWhitelist(uint64 _expiryTime) internal view returns(bool) {
+        return (_expiryTime >= uint64(now)); /*solium-disable-line security/no-block-members*/
```

Comment:
Do we need this function anymore? it's just `_expiryTime >= uint64(now)`. Calling and defining a function takes gas.

Reply:
At the very least, we should rename this function.

Reply:
Aren't all internal functions just in-lined anyway? Having this function (and corresponding _validLockTime) makes the code marginally more readable IMO.

Reply:
internal functions of libraries are inlined but normal internal functions of a contract are not inlined. they are treated like normal functions.

The change won't be saving much gas anyway. we can keep the function.

Reply:
A small test in Remix shows a very small gas difference ~50 gas between using an internal function and manually in-lining the code. Given the tiny difference I think readability wins here.